src/codemodder/codemods/utils.py

```diff
@@ -192,3 +192,27 @@ def is_assigned_to_True(original_node: cst.Assign):
         isinstance(original_node.value, cst.Name)
         and original_node.value.value == "True"
     )
+
+
+def is_zero(node: cst.CSTNode) -> bool:
```

Comment:
This is pretty cool! I do wonder whether `ast.literal_eval` would work just as well for these purposes, but I'm okay with this.

Reply:
I tested it out but it only works well for the `case cst.Integer() | cst.Float():`, everything else still requires special casing manually so yeah gonna leave it as is for now. Good call though!